### ISSUE TITLE
The fix is not permanent when using the dispwin utility.

### ISSUE THREAD

**Issue by Mario0051** [dylanraga/win11hdr-srgb-to-gamma2.2-icm#68]
The dispwin utility seems to apply the fix, but it gets reverted almost immediately as if the revert.bat script was ran.

**Comment by Manticore425** [comment]
Yes, same question

**Comment by Animesh-Does-Code** [comment]
This is a shot in the dark, but do you have DisplayCal installed and running? Try closing it.

I think it has a feature that re-applies Windows color profiles, which will revert any changes made by the dispwin utility.

**Comment by Mario0051** [comment]
No, it is not running at all, and not installed since I have a portable version.

**Comment by Manticore425** [comment]
After some trying, I decided that the simplest solution was to write a script to monitor the loading of lut.cal in real time. Although I do not know the reason of the revert, this has certainly solved my problem. Example(powershell ps1):
```
while ($true) {
    # [System.Console]::Clear()
    cls
    $output = & .\dispwin.exe -V .\lut.cal 2>&1
    Write-Host "Current state: $output"
    
    if ($output -match "is NOT loaded") {
        Write-Host "lut.cal not loaded!"
        & .\dispwin.exe .\lut.cal
        Write-Host "OK! "
    }
    else {
        Write-Host "Well done!"
    }
    Start-Sleep -Seconds 15
}
```

**Comment by Mario0051** [comment]
The script does reactivate the color profile, but it can noticeably stay reverted for the whole 15 second time period.

**Comment by Manticore425** [comment]
Then you have a bigger problem than I do. My computer reverts after a few tens of seconds of execution, not immediately, so this simple script can achieve full coverage. Try to update Argyll's dispwin?

**Comment by Mario0051** [comment]
The V3.3.0 variant of dispwin also exhibits the same issue, unfortunately.

**Comment by Mario0051** [comment]
I found that opening the notification tray will instantly revert it, try it as a test.

**Comment by netExtra** [comment]
Win11 25H2. I ran your script slightly modified so the screen doesn't clear. Opening notification or system tray didn't make any difference:

Current state: Verify: '.\lut.cal' IS loaded (discrepancy 0.0%)
Well done!
Current state: Verify: '.\lut.cal' IS loaded (discrepancy 0.0%)
Well done!
Current state: Verify: '.\lut.cal' IS loaded (discrepancy 0.0%)
Well done!
Current state: Verify: '.\lut.cal' IS loaded (discrepancy 0.0%)
Well done!
Current state: Verify: '.\lut.cal' IS loaded (discrepancy 0.0%)
Well done!
Current state: Verify: '.\lut.cal' IS loaded (discrepancy 0.0%)
Well done!
Current state: Verify: '.\lut.cal' IS loaded (discrepancy 0.0%)
Well done!
Current state: Verify: '.\lut.cal' IS loaded (discrepancy 0.0%)
Well done!
Current state: Verify: '.\lut.cal' IS loaded (discrepancy 0.0%)
Well done!
Current state: Verify: '.\lut.cal' IS loaded (discrepancy 0.0%)
Well done!
Current state: Verify: '.\lut.cal' IS loaded (discrepancy 0.0%)
Well done!
.... and it's still going :)
